.travis.yml

```diff
@@ -14,13 +14,26 @@ matrix:
       dist: xenial
       sudo: true
 
+env:
+  global:
+    # Doctr deploy key for coyo8/sec-edgar
+    - secure: "Yli1M7XnE5EmLhSkfSajLelrNs9rjxcMT59e7G7586OaOdzgEviJA285HCChwJdkfYNyoa3msMwx/ciQHn6tQ7xWm8EowG4v+L7WVgFZ9MacdG4nYqOrDIKfQUpK64rfbD5GGlTd8ptmgCN2+RtY8xkJ1Ajbbb7GqBc2XeSDTbzrCp8feIkV9gV3Nvcz7O4g1zBuyiuR+w9fO66rzcVEvqraZWavt5mrMqB1T8ghqWBGYGgnL3NXBcbdkgyaD8IPtR9twpGc3x1b9jwzQdYKBZ4bvbNwn3qYQwkFkXOt9rBZN+Ba70vKwGGptI10QPCrkxkYGM6HkcX8lTPJyhAFJ4iIGqkQ/kC63SHXgQ0rEBTwu2wXQ835dYJ5H83718mDkQ3jO03ucKFq343oQJcJfdIiVV2YOQamTXlqtpaJ6kpGk94rjrWa0INPl8PJ8lrowP1CrDEIOS20T6hUtGI3dzf/5Ei1nGluuScOobFJSDAg4ea0aTOjktmOcVyruCqpjdF74GnoLIkij96oPE7t/ear4CQjUM5VICSYDJL9kBEg+ptTsIeJE73KSc/SnVT13maf/HVpav6xdKWrwQBxkmdo4Nwe8LFWDD3wQmcICJpY0mxrTqwvpCRFPBhJMciJdJPp7Z+IiHGvoJ0eT8j6ERZL0hdaUV4VfUq9aA5L4gk="
 
 install:
-  - pip install -qq flake8
-  - pip install pytest
+  - pip install -r requirements-dev.txt
   - python setup.py install
 
 script:
   - flake8 --version
   - flake8 SECEdgar
   - pytest SECEdgar/tests
+
```

Comment:
I like how @addisonlynch does this in his iexfinance package. You can check it out [here](https://github.com/addisonlynch/iexfinance/blob/master/.travis.yml). See my suggestions below. I made some slight changes relevant for our project.
```
include:
    - python: 3.6
      dist: trusty
      env: DOCBUILD=true
```

```
install:
  - pip install -qq flake8
  - pip install pytest
  - if [[ $DOCBUILD ]]; then
      pip install doctr sphinx sphinx_rtd_theme ipython matplotlib
    fi
  - python setup.py install

script:
  - set -e
  - flake8 --version
  - flake8 SECEdgar
  - pytest SECEdgar/tests
  - |
    if [[ $DOCBUILD]]; then
      cd docs
      make html
      cd ..
      doctr deploy devel --build-tags
      if [[ -z ${TRAVIS_TAG} ]]; then
        echo "Not a tagged build."
      else
        doctr deploy stable --build-tags
      fi
    fi
```